src/Entity/Activity/Activity.php

```diff
@@ -429,7 +434,7 @@ public function setCapacity(?int $capacity): self
         return $this;
     }
 
-    public function getPresent()
+    public function getPresent(): ?int
```

Comment:
why is this an int and not a bool. I get the 0=unknows, 1=present, 2=not present.
but we could make it an optional field and leave it null/empty on onknown. or is there logic hanging on it being an int?

if so..... do we really want it that way?

Reply:
Present is a number, Aka: 150 people were present during this activity. If it is null however: No amount of present was set manually and the number is then real time automaticaly generated, based on individual registration Present.

Reply:
get Present then seems a little counter intuitive, that could be just me.
@jasperweyne thoughts?